app/models/community.rb

```diff
@@ -145,8 +145,8 @@ class Community < ActiveRecord::Base
   VALID_BROWSE_TYPES = %{grid map list}
   validates_inclusion_of :default_browse_view, :in => VALID_BROWSE_TYPES
 
-  VALID_NAME_DISPLAY_TYPES = %{first_name_only first_name_with_initial}
-  validates_inclusion_of :default_browse_view, :in => VALID_BROWSE_TYPES, :allow_nil => true, :allow_blank => true
+  VALID_NAME_DISPLAY_TYPES = %{first_name_only first_name_with_initial full_name}
+  validates_inclusion_of :name_display_type, :in => VALID_NAME_DISPLAY_TYPES
```

Comment:
:default_browse_view has validations just above these lines. 
(So this is just old copy-paste error, which has not worked correctly ever)
As discussed irl, I started to check if this has real-life effects since there was before allow_nil etc. extra rules ...deciced not to spend time with that since we should never allow nil or blank for this parameter.

This is related to
http://localhost:3000/en/admin/communities/1/edit_look_and_feel#community_default_browse_view
Where there are just 3 options allowed (and unlike previous name-type selection functionality, there is no blank as default value for any of those three options. And after this bugfix, blank option is removed from name-type select too.)